otelcollector/configmaps/ama-metrics-settings-configmap.yaml

```diff
@@ -81,7 +81,8 @@ data:
       enabled = true
   prometheus-collector-settings: |-
     cluster_alias = ""
-    debug-mode = "false"
+    debug-mode = false
```

Comment:
Does changing this from string to bool have any effect on the parsing?

Reply:
yes, it did have an effect in the older version where this **had** to be a bool, but @bragi92 fixed this in the latest version, thats a good test case too. I will add it in the onenote